<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>VAGA</class>
 <widget class="QDialog" name="VAGA">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>VAGA - Volatility GUI Analyzer</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icon.ico</normaloff>icon.ico</iconset>
  </property>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>80</y>
     <width>1150</width>
     <height>370</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="file_path">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>421</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="select">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Select</string>
   </property>
  </widget>
  <widget class="QPushButton" name="analyze">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>10</y>
     <width>75</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Analyze</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pslist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>pslist</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.ico</normaloff>icon.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="psscan">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>110</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>psscan</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.ico</normaloff>icon.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="info">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>170</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>info</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.ico</normaloff>icon.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="cmdline">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>cmdline</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.ico</normaloff>icon.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="dlllist">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>230</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>dlllist</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.ico</normaloff>icon.ico</iconset>
   </property>
  </widget>
  <widget class="QPushButton" name="pstree">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>140</y>
     <width>75</width>
     <height>24</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>pstree</string>
   </property>
   <property name="icon">
    <iconset>
     <normaloff>icon.ico</normaloff>icon.ico</iconset>
   </property>
  </widget>
  <widget class="QLabel" name="log_report">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>641</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Select DataBase File!!</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>500</y>
     <width>1281</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
